<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 0, 0)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color: rgb(1, 255, 1);</string>
   </property>
   <widget class="QFrame" name="frameleft">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>181</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnDashboard">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-position: top left;
    border-image url(:/newPrefix/BrokenSword.ico) 1 100 0 0 stretch stretch;
	background-repeat: no-reperat;
	border: none;
	background-color: rgb(0,0,0);
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="main.qrc">
       <normaloff>:/newPrefix/BrokenSword.ico</normaloff>:/newPrefix/BrokenSword.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFormular">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-position: top left;
    border-image url(:/newPrefix/BrokenSword.ico) 1 100 0 0 stretch stretch;
	background-repeat: no-reperat;
	border: none;
	background-color: rgb(0,0,0);
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}url(:/newPrefix/atom-beta.ico)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="main.qrc">
       <normaloff>:/newPrefix/atom-beta.ico</normaloff>:/newPrefix/atom-beta.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSettings">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>530</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-position: top left;
    border-image url(:/newPrefix/setroubleshoot_icon.ico)1 100 0 0 stretch stretch;
	background-repeat: no-reperat;
	border: none;
	background-color: rgb(0,0,0)
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}url(:/newPrefix/atom-beta.ico)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="main.qrc">
       <normaloff>:/newPrefix/setroubleshoot_icon.ico</normaloff>:/newPrefix/setroubleshoot_icon.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lblcapitalmanager">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(88, 1, 200);</string>
     </property>
     <property name="text">
      <string>Capital Manager</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCompanyFormular">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-position: top left;
    border-image url(:/newPrefix/BrokenSword.ico) 1 100 0 0 stretch stretch;
	background-repeat: no-reperat;
	border: none;
	background-color: rgb(0,0,0);
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}url(:/newPrefix/atom-beta.ico)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="main.qrc">
       <normaloff>:/newPrefix/flare.ico</normaloff>:/newPrefix/flare.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="framdown">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>619</y>
      <width>1051</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 85, 55)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frmDashboard">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>871</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(50, 0, 0)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>281</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>frmDashboard</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmFormular">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>871</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,50, 0)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblSpendMoney">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(88, 1, 200);</string>
     </property>
     <property name="text">
      <string>Spend Money</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbspendmoney">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(10,10,10);
color: rgb(88, 1, 200);</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>Main Account</string>
     </property>
     <item>
      <property name="text">
       <string>Main Account</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Account 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Account 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Account 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Account 4</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnSend_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>540</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 2px solid rgb(37,39,48);
border-radius: 10px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(34,36,44);
color:rgb(0,0,255);
}</string>
     </property>
     <property name="text">
      <string>Get</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="pbMainAccount">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	
	background-color: rgb(98, 114, 164);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
}</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="pbAccount2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>410</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	
	background-color: rgb(98, 114, 164);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
}</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="pbAccount3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>440</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	
	background-color: rgb(98, 114, 164);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
}</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="pbAccount4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>470</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	
	background-color: rgb(98, 114, 164);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
}</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="pbAccount1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>380</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	
	background-color: rgb(98, 114, 164);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
}</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblMainAccount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>345</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(88, 1, 200);</string>
     </property>
     <property name="text">
      <string>Main Account</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAccount1">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>380</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(88, 1, 200);</string>
     </property>
     <property name="text">
      <string>Account 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAccount2">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>410</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(88, 1, 200);</string>
     </property>
     <property name="text">
      <string>Account 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAccount4">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>470</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(88, 1, 200);</string>
     </property>
     <property name="text">
      <string>Account 4</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAccount3">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>440</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(88, 1, 200);</string>
     </property>
     <property name="text">
      <string>Account 3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lespend">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>172</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 10px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(34,36,44);
color:rgb(0,0,255);
}
QLineEdit:hover { border: 2px solid rgb(48,50,62);}QLineEdit:focus{border:2px solid rgb(48,50,60)n}</string>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSpend">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>171</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 2px solid rgb(37,39,48);
border-radius: 10px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(34,36,44);
color:rgb(0,0,255);
}</string>
     </property>
     <property name="text">
      <string>Spend</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(88, 1, 200);</string>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="pbTotal">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>523</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	
	background-color: rgb(98, 114, 164);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(44, 2, 100, 255));
}</string>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmCompanyFormular">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>871</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0, 0)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>751</x>
       <y>0</y>
       <width>120</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 45, 0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>580</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>lblcompany</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(10,10,10);
color: rgb(88, 1, 200);</string>
     </property>
     <item>
      <property name="text">
       <string>Account_991050</string>
      </property>
     </item>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>751</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>45</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 10px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(34,36,44);
color:rgb(0,0,255);
}
QLineEdit:hover { border: 2px solid rgb(48,50,62);}QLineEdit:focus{border:2px solid rgb(48,50,60)n}</string>
      </property>
      <property name="text">
       <string>3000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(88, 1, 200);</string>
      </property>
      <property name="text">
       <string>Amount:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSend">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>470</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: 2px solid rgb(37,39,48);
border-radius: 10px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(34,36,44);
color:rgb(0,0,255);
}</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </widget>
    <zorder>frame_2</zorder>
    <zorder>frame</zorder>
    <zorder>label_3</zorder>
    <zorder>comboBox_2</zorder>
   </widget>
   <widget class="QFrame" name="frametop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>871</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 40, 20)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblname">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>0</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(88, 1, 200);</string>
     </property>
     <property name="text">
      <string>Dashboard</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>frmDashboard</zorder>
   <zorder>frameleft</zorder>
   <zorder>framdown</zorder>
   <zorder>frametop</zorder>
   <zorder>frmCompanyFormular</zorder>
   <zorder>frmFormular</zorder>
  </widget>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
